public/partial/recipe-detail-main.html

```diff
@@ -57,24 +57,24 @@
     <ng-include src="'partial/recipe-detail-indicators.html'"></ng-include>
     <ng-include src="'partial/recipe-detail-operations.html'"></ng-include>
     <div class="panel panel-default">
-        <div class="panel-heading"><strong>Nazi Mode</strong></div>
+        <div class="panel-heading"><strong>Modo estricto</strong></div>
```

Comment:
This is the only change you have to make.

Reply:
> This is the only change you have to make.

Addressed in last commit